lib/app.coffee

```diff
@@ -33,6 +33,9 @@ events = require('./events')
 plugins = require('./utils/plugins')
 update = require('./utils/update')
 
+# Assign bluebird as the global promise library
+require('any-promise/register/bluebird')
```

Comment:
It's for `streamToPromise`, otherwise it uses native promises and messes things up, a lot.